tests/jballerina-unit-test/src/test/resources/test-src/types/xml/xmlns-def.bal

```diff
@@ -0,0 +1,20 @@
+const A = "http://ballerina.com/a";
+
+xmlns A as ns0;
```

Comment:
Shall we add a test where const A is union type(one member is string). Shall we add a negative test where A is not a string type?